presto-parser/src/main/java/com/facebook/presto/sql/SqlFormatter.java

```diff
@@ -381,8 +382,14 @@ protected Void visitAliasedRelation(AliasedRelation node, Integer indent)
         {
             process(node.getRelation(), indent);
 
-            builder.append(' ')
-                    .append(node.getAlias());
+            builder.append(' ');
+            String alias = node.getAlias();
```

Comment:
Use `formatName()` for this. It handles anything that is not a valid un-quoted identifier.